### DISCUSSION TITLE
Performance tests

### DISCUSSION THREAD

**Discussion by Xact** [rabbibotton/clog#61] [General]
Hi! Looking for suggestions on how to test clog app performance. My little demo is blipping short client-side audio samples at a rate of 500 bpm at which point everything gets very hot and starts breaking apart at the seams. I thought pushing all audio to JavaScript would fix things but I do have threads on clog side too. So anyway, other than using browser debug/performance tools, does anyone have suggestions? Maybe a page of profiling/performance advice for clog users would be a good idea? (Here’s an example that node.js have https://nodejs.org/en/docs/guides/simple-profiling/)

**Comment by rabbibotton** [comment]
I currently have not been working on that side of things, but will make a point to eventually.
